### PR TITLE
Feature/settings

### PR DESCRIPTION
**Pull request by hvandermerwe**
fixed redux on the settings page. Users are now able to edit the username and user email address

### CODE REVIEW

File: chatterbox/src/Main/MainArea/Settings/user-settings-component.js [hunk 1]
```diff
@@ -20,56 +22,115 @@ const SubmitChanges =  {
...
+            </div>
+        );
+    }
+
+    renderEditName() {
+        return (
+            <div>
+                <div className="user-name">
+                    <TextField  
+                        defaultValue={this.props.user.user.name}
```
Comment (KimberleighBeyl) [inline chatterbox/src/Main/MainArea/Settings/user-settings-component.js]: user.user.name?

File: chatterbox/src/Main/MainArea/Settings/user-settings-component.js [hunk 2]
```diff
@@ -20,56 +22,115 @@ const SubmitChanges =  {
...
+                        onClick={() => this.props.editUserEmail(this.props.user.user)}
+                        className="edit"
+                    />
+                </div>
+            </div>
+        );
+    }
+
     render() {
+        console.log(this.props)
```
Comment (KimberleighBeyl) [inline chatterbox/src/Main/MainArea/Settings/user-settings-component.js]: remove this line

File: chatterbox/src/modules/settings/user-actions.js [hunk 3]
```diff
@@ -1,34 +1,31 @@
 export const editUserName = (user) => {
+    console.log("Edit user name " + user.name)
```
Comment (KimberleighBeyl) [inline chatterbox/src/modules/settings/user-actions.js]: remove

File: chatterbox/src/modules/settings/user-actions.js [hunk 4]
```diff
@@ -1,34 +1,31 @@
...
     return {
         type: "EDIT_USER_NAME",
         payload: user
     }
 }
 
 export const editUserEmail = (user) => {
+    console.log("Edit user email " + user.email)
     return {
         type: "EDIT_USER_EMAIL",
```
Comment (KimberleighBeyl) [inline chatterbox/src/modules/settings/user-actions.js]: remove

**Review by KimberleighBeyl — CHANGES_REQUESTED** [review]
remove all console logs and just check this.props.user.user

File: chatterbox/src/Main/MainArea/Settings/user-settings-component.js (hunk 1 above)
Comment (hvandermerwe) [reply chatterbox/src/Main/MainArea/Settings/user-settings-component.js]: Thank you for pointing that out. it has been fixed

File: chatterbox/src/Main/MainArea/Settings/user-settings-component.js (hunk 2 above)
Comment (hvandermerwe) [reply chatterbox/src/Main/MainArea/Settings/user-settings-component.js]: I removed the console.log()

File: chatterbox/src/modules/settings/user-actions.js (hunk 3 above)
Comment (hvandermerwe) [reply chatterbox/src/modules/settings/user-actions.js]: I removed the console.log()

File: chatterbox/src/modules/settings/user-actions.js (hunk 4 above)
Comment (hvandermerwe) [reply chatterbox/src/modules/settings/user-actions.js]: it has been removed

**Review by KimberleighBeyl — APPROVED** [review]
LGTM

**Pull request merged**